reference/configuration/framework.rst

```diff
@@ -526,10 +526,10 @@ enabled
 
 Whether or not to enable the ``translator`` service in the service container.
 
-fallback
-........
+fallbacks
+.........
 
-**type**: ``string`` **default**: ``en``
+**type**: ``array`` **default**: ``en``
```

Comment:
Should we also explain that the old option name still works so that people don't get worried about BC breaks when reading this?